src/main/java/org/jabref/gui/importer/ImportEntriesViewModel.java

```diff
@@ -52,6 +55,7 @@
     private final PreferencesService preferences;
     private final BibEntryTypesManager entryTypesManager;
 
+    private final ImportCleanup cleanup = new ImportCleanup(BibDatabaseMode.BIBTEX);
```

Comment:
I guess these were changes from previous commits?

Reply:
Somehow my branch was not up to date.